apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/lexer/ApexLexer.java

```diff
@@ -26,7 +26,7 @@
     /**
      * Stores a pattern to identify a keyword.
      */
-    private static final String IDENTIFIER_PATTERN = "(_{0,2}[a-zA-Z][a-zA-Z0-9]*)+";
+    private static final String IDENTIFIER_PATTERN = "(_{0,2}[a-zA-Z][_a-zA-Z0-9]*)+";
```

Comment:
@JimmyRojas for example, this case '_myVar_001' should not pass and indeed it doesn't, but I think the issue is it supports any number of underscores, do you think I should control that somehow?